alf/tensor_specs.py

```diff
@@ -218,10 +255,12 @@ def is_bounded(cls):
 
     @classmethod
     def from_spec(cls, spec):
-        assert isinstance(spec, BoundedTensorSpec)
-        minimum = getattr(spec, "minimum")
-        maximum = getattr(spec, "maximum")
-        return BoundedTensorSpec(spec.shape, spec.dtype, minimum, maximum)
+        assert isinstance(spec, (TensorSpec, BoundedTensorSpec))
+        if hasattr(spec, 'minimum') and hasattr(spec, 'maximum'):
+            return BoundedTensorSpec(spec.shape, spec.dtype, spec.minimum,
+                                     spec.maximum)
+
+        return BoundedTensorSpec(spec.shape, spec.dtype)
```

Comment:
There are always default `min` and `max` values for `BoundedTensorSpec`. So you don't need `if`